src/components/SearchInput/SearchInput.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import classnames from 'classnames'
+
+import { Button } from '../Button/Button'
+import { Form } from '../forms/Form/Form'
+import { Label } from '../forms/Label/Label'
+import { TextInput } from '../forms/TextInput/TextInput'
+
+interface SearchInputProps {
+  onSubmit: (event: React.FormEvent<HTMLFormElement>) => void
+  className?: string
+}
+
+export const SearchInput = (props: SearchInputProps): React.ReactElement => {
+  const { onSubmit, className } = props
+  const classes = classnames('usa-search usa-search--small', className)
```

Comment:
Do you mind implementing the 3 styles (default, `big`, `small`) described at https://designsystem.digital.gov/components/search/ ? I think that will involve toggling different CSS classes and a slight markup difference for the Search button text